### PR TITLE
Update discovery to 16.1

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly
* 2.1

### CODE REVIEW

File: packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec [hunk 1]
```diff
@@ -6,8 +6,8 @@
 %global plugin_name discovery
 %global foreman_min_version 1.24.0
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec:7]: Is this still correct?

**Comment by evgeni** [comment]
hah, ewoud cought a good point

File: packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec (hunk 1 above)
Comment (lzap) [reply packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec:7]: Technically yes, I haven't tried tho. But I don't see any incompatibility.

**Comment by lzap** [comment]
I used the tool and it did just this:

```diff
diff --git a/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.0.1.gem b/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.0.1.gem
deleted file mode 120000
index bb8879726..000000000
--- a/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.0.1.gem
+++ /dev/null
@@ -1 +0,0 @@
-../../../.git/annex/objects/1F/xx/SHA256E-s314880--a33fca1b9d5d2abedb958f04d0c6cac03528296fe15f17bf2754eb6aa893a365.1.gem/SHA256E-s314880--a33fca1b9d5d2abedb958f04d0c6cac03528296fe15f17bf2754eb6aa893a365.1.gem
\ No newline at end of file
diff --git a/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.1.0.gem b/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.1.0.gem
new file mode 120000
index 000000000..340eaae93
--- /dev/null
+++ b/packages/plugins/rubygem-foreman_discovery/foreman_discovery-16.1.0.gem
@@ -0,0 +1 @@
+../../../.git/annex/objects/k6/42/SHA256E-s312320--d3a6387ed6cce4ba731125844a7fe6ca2505a186c841a6fcdc9f7f04a85c01c2.0.gem/SHA256E-s312320--d3a6387ed6cce4ba731125844a7fe6ca2505a186c841a6fcdc9f7f04a85c01c2.0.gem
\ No newline at end of file
diff --git a/packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec b/packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec
index 40a540935..f96cead7b 100644
--- a/packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec
+++ b/packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.1.0
 Release:    %{?prever:0.}%{release}%{?prever}%{?foremandist}%{?dist}
 Group:      Applications/Systems
 License:    GPLv3
@@ -105,6 +105,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/test
 
 %changelog
+* Wed May 27 2020 Lukas Zapletal <lzap+rpm@redhat.com> 16.1.0-2
+- Update to 16.1.0
+
 * Tue Jan 07 2020 Eric D. Helms <ericdhelms@gmail.com> - 16.0.1-2
 - Drop migrate, seed and restart posttans
```

That does not look correct. Teach me!

**Comment by evgeni** [comment]
@lzap I think we spoke about that in the past, your rpm spec is not using our template, and thus our tools fail for you. I'd merge this as-is, but let's maybe go and standartize the spec in the 2.2 devel cycle?

**Pull request merged**